src/components/shared/Stats.tsx

```diff
@@ -1,13 +1,29 @@
 import '../../styles/stats.scss';
+
+type statsObject = {
```

Comment:
I would make a playerStats object somewhere and use it both here and in the state of App.tsx